internal/renameio/renameio_test.go

```diff
@@ -130,10 +128,18 @@ func TestConcurrentReadsAndWrites(t *testing.T) {
 	}
 
 	var minReadSuccesses int64 = attempts
-	if runtime.GOOS == windowsOS {
+
+	switch runtime.GOOS {
+	case "windows":
 		// Windows produces frequent "Access is denied" errors under heavy rename load.
-		// As long as those are the only errors and *some* of the writes succeed, we're happy.
+		// As long as those are the only errors and *some* of the reads succeed, we're happy.
 		minReadSuccesses = attempts / 4
+
+	case "darwin":
+		// The filesystem on macOS 10.14 occasionally fails with "no such file or
```

Comment:
😞 why can't we have nice things?

I like the "robustio" package though (interesting)

(Also, sorry for the noise; just glancing over the code changes, and learned about this macOS bug)

Reply:
Yep.
Also, it's copy-pasted code from [Go source](https://github.com/golang/go/tree/master/src/cmd/go/internal/renameio)